crates/astria-composer/src/executor/bundle_factory/mod.rs

```diff
@@ -24,17 +33,34 @@ enum SizedBundleError {
     SequenceActionTooLarge(SequenceAction),
 }
 
+pub(super) struct RollupCountsReport(HashMap<RollupId, u32>);
+
+impl Serialize for RollupCountsReport {
```

Comment:
Does this need a manual `Serialize` impl? We need it in conductor because we construct information ad-hoc, but I think in this case you might get away with something like this (since `RollupId` should impl `Serialize`; you might need to adjust the features of `astria-core`):
```rust
#[derive(serde::Serialize)]
#[serde(transparent)]
pub(super) struct RollupCountsReport(HashMap<RollupId, u32>);
```
Less work